src/internal/connector/sharepoint/restore.go

```diff
@@ -76,6 +76,8 @@ func RestoreCollections(
 				deets,
 				errUpdater,
 			)
+		case path.PagesCategory:
```

Comment:
putting this here as it has nicer thread support than just comments in the issue in my opinion

I'm a little out of the loop on the SharePoint stuff, but do we expect to release with only partial SharePoint restore support? I know we'll get everything eventually, but I haven't been following this closely enough to know what the plan is right now

Reply:
Yes. In the case of Pages, it is `unusable` until the Beta library can be utilized. Whether this is through creating a separate binary or submodule for beta, it is unclear how it would be incorporated. Lists can be downloaded and restored at this time. As for Libraries, there is a need either create a Corso workflow that would allow for items within restricted directories to be uploaded and migrated to their correct locations.

Reply:
I just a bit confused as this code is in the restore path, not the backup path. I get that we're eventually going to be pulling in the beta version to access more Graph API functionality

Reply:
Title/comments oriented appropriately.